composeApp/src/commonMain/kotlin/com/github/familyvault/services/FamilyGroupService.kt

```diff
@@ -115,6 +116,24 @@ class FamilyGroupService(
         )
     }
 
+    override suspend fun changeFamilyMemberPermissionGroupToGuardian(userId: String) {
+        familyVaultBackendProxy.changeFamilyMemberPermissionGroup(
+            ChangeFamilyMemberPermissionGroupRequest(familyGroupSessionService.getContextId(), userId, FamilyGroupMemberPermissionGroup.Guardian)
+        )
+    }
+
+    override suspend fun changeFamilyMemberPermissionGroupToMember(userId: String) {
+        familyVaultBackendProxy.changeFamilyMemberPermissionGroup(
+            ChangeFamilyMemberPermissionGroupRequest(familyGroupSessionService.getContextId(), userId, FamilyGroupMemberPermissionGroup.Member)
+        )
+    }
+
+    override suspend fun changeFamilyMemberPermissionGroupToGuest(userId: String) {
+        familyVaultBackendProxy.changeFamilyMemberPermissionGroup(
+            ChangeFamilyMemberPermissionGroupRequest(familyGroupSessionService.getContextId(), userId, FamilyGroupMemberPermissionGroup.Guest)
+        )
+    }
+
```

Comment:
Niezła kaszana się zaczyna robić w `FamilyGroupService`, wydzielmy to do `FamilyMemberPermissionGroupService`